test/integration/targets/setup_podman/vars/RedHat-8.yml

```diff
@@ -1 +1 @@
-podman_package: '@container-tools:1.0'
+podman_package: podman
```

Comment:
Can you please do the same for rhel7?

Reply:
I need to see what version is available for RHEL 7. It's currently `1.3.*` for RHEL 7.

Reply:
@samdoran  podman.x86_64 0:1.4.4-4.el7 according to https://app.shippable.com/github/ansible/ansible/runs/150825/75/console